### PR TITLE
Update Desktop view in Desktop branch

### PR DESCRIPTION
**Pull request by ohyajk**
I added HTML, CSS and asset Images to make website portfolio responsive using media queries.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -414,3 +429,293 @@ form {
...
+  }
+
+  .btn-a {
+    display: flex;
+    justify-content: start;
+  }
+
+  .btn {
+    width: 45%;
+  }
```
Comment (christianbiring1) [inline style.css:531]: - [ ] Kindly consider implementing the button interaction style as it is shown on the Figma template when the user `hover, press and disable`
Figma template :point_down: 
image

File: style.css [hunk 2]
```diff
@@ -414,3 +429,293 @@ form {
...
+  body {
+    margin: 0;
+    padding: 0;
+    background-image: none;
+    overflow: hidden;
+    overflow-y: scroll;
+  }
+
+  .menu {
+    display: none;
```
Comment (christianbiring1) [inline style.css:456]: - [ ] Kindly make sure your page is responsive on the screen size of `768px`. The element should be centered as they are for another screen size. Kindly fix that.
Your design :point_down: 
Screenshot from 2022-11-03 08-34-15

Screenshot from 2022-11-03 08-34-35

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @ohyajk , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `sumary`
- [ ] Good UI :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @ohyajk, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Good commit messages 👍 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**